tutorial/cache_pyh_files.md

```diff
@@ -0,0 +1,32 @@
+# Picklize Header files
```

Comment:
I think that we should name this file `headers.md`, and change the title to `Pyccel header files`. We can then skip the first subtitle altogether.

Reply:
we already have another file named `headers.md`

Reply:
Ah, that's true! But that is about function headers and decorators, not about header files.

Maybe this file should be named `header_files.md` then?

Reply:
okay